app/views/templates/_form.html.erb

```diff
@@ -84,6 +83,70 @@
       <% end %>
     </div>
 
+    <div class="tab-pane" id="help">
+      <div>
+        <%= _('Provisioning templates use ERB syntax') %>
```

Comment:
It ought to be a sentence, but I struggle to figure out something reasonable :-) Will try..,.